magda-semantic-indexer-framework/src/skipError.ts

```diff
@@ -0,0 +1 @@
+export class SkipError extends Error {}
```

Comment:
Probably want to match the file name case with the class name i.e. the first letter of the file name should be upper case as well